spec/requests/cloud_volume_types_spec.rb

```diff
@@ -0,0 +1,44 @@
+#
+# REST API Request Tests - Cloud Volume Types
+#
+# Regions primary collections:
+#   /api/cloud_volume_types
+#
+# Tests for:
+# GET /api/cloud_volume_types/:id
+#
+
+describe "Cloud Volume Types API" do
+  it "forbids access to cloud volume types without an appropriate role" do
+    api_basic_authorize
+    get(api_cloud_volume_types_url)
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "allows access to cloud volume types with an appropriate role" do
+    api_basic_authorize action_identifier(:cloud_volume_types, :read, :collection_actions, :get)
+    instance = CloudVolumeType.create!
+    get(api_cloud_volume_types_url)
+    expect(response).to have_http_status(:ok)
+    url = api_cloud_volume_type_url(nil, instance)
```

Comment:
This test should be hitting `api_cloud_volume_types_url` with no `instance`

Reply:
it is - see the `get` statement

This line and the next is to verify that a resource in the collection is returned